source/backend/api/Areas/CompensationRequisition/Controllers/CompensationRequisitionController.cs

```diff
@@ -85,33 +110,54 @@ public IActionResult UpdateCompensationRequisition([FromRoute] long id, [FromBod
                 throw new BadRequestException("Invalid compensationRequisitionId.");
             }
 
-            var compensationReqEntity = _mapper.Map<Dal.Entities.PimsCompensationRequisition>(compensationRequisition);
+            var compensationReqEntity = _mapper.Map<PimsCompensationRequisition>(compensationRequisition);
             var compensation = _compensationRequisitionService.Update(compensationReqEntity);
```

Comment:
It feels odd that we use filetypes for add but not for update. Why is that?

Reply:
the Filetype is not needed to update or delete a compensation requisition. I could pass the filetype as a parameter but at the moment is needed for any validation.

Reply:
Technically is not needed but I could add it.

Reply:
Added